docs/contributions/online-contributions.md

```diff
@@ -150,12 +150,13 @@ To add a profile to a contribution form:
 
     If you include a profile with address fields at the top of the page, CiviCRM
     automatically generates a checkbox on the contribution form which allows the
-    user to indicate that their Billing Address is the same as the address
-    entered in the profile. (If the profile is included at the bottom of the
-    page, this checkbox will not be generated).
-
+    user to indicate that his/her Billing Address is the same as the address
```

Comment:
We aim to use gender neutral language - we won’t accept PRs that reintroduce “his/her”over “their”.